src/db/schema.ts

```diff
@@ -11,7 +11,7 @@ import {
 } from "drizzle-orm/pg-core"
 
 export const roleEnum = pgEnum("role", ["member", "admin"])
-export const betResultEnum = pgEnum("bet_results", ["win", "loss"])
+export const betResultEnum = pgEnum("bet_result", ["win", "loss", "tie"]) // เพิ่ม "tie"
```

Comment:
ขอปรับ tie เป็น loss ไปเลยถ้าแก้เพิ่มมันจะเสียทั้ง db อ่ะ

Reply:
แก้แล้ว
[fix: use betResult loss instead of tie](https://github.com/beambeambeam/modbid888/pull/36/commits/1a80558ad885a49f7426e29a1fc5133ea8b262db)